checkout-example/views/index.handlebars

```diff
@@ -12,96 +12,47 @@
-      <a href="/preview?type=card" class="integration-list-item-link">
-        <div class="title-container">
-          <p class="integration-list-item-title">Card</p>
-        </div>
-      </a>
-    </li>
-    <li class="integration-list-item">
-      <a href="/preview?type=ideal" class="integration-list-item-link">
-        <div class="title-container">
-          <p class="integration-list-item-title">iDEAL</p>
-        </div>
-      </a>
-    </li>
-    <li class="integration-list-item">
-      <a href="/preview?type=dotpay" class="integration-list-item-link">
-        <div class="title-container">
-          <p class="integration-list-item-title">Dotpay</p>
-        </div>
-      </a>
-    </li>
-    <li class="integration-list-item">
-      <a href="/preview?type=giropay" class="integration-list-item-link">
-        <div class="title-container">
-          <p class="integration-list-item-title">giropay</p>
-        </div>
-      </a>
-    </li>
-    <li class="integration-list-item">
-      <a href="/preview?type=sepadirectdebit" class="integration-list-item-link">
-        <div class="title-container">
-          <p class="integration-list-item-title">SEPA Direct Debit</p>
-        </div>
-      </a>
-    </li>
-    <li class="integration-list-item">
-      <a href="/preview?type=directEbanking" class="integration-list-item-link">
-        <div class="title-container">
-          <p class="integration-list-item-title">SOFORT</p>
-        </div>
-      </a>
-    </li>
-    <li class="integration-list-item">
-      <a href="/preview?type=ach" class="integration-list-item-link">
-        <div class="title-container">
-          <p class="integration-list-item-title">ACH</p>
-        </div>
-      </a>
-    </li>
-    <li class="integration-list-item">
-      <a href="/preview?type=alipay" class="integration-list-item-link">
-        <div class="title-container">
-          <p class="integration-list-item-title">Alipay</p>
-        </div>
-      </a>
-    </li>
-    <li class="integration-list-item">
-      <a href="/preview?type=klarna_paynow" class="integration-list-item-link">
-        <div class="title-container">
-          <p class="integration-list-item-title">Klarna - Pay now</p>
-        </div>
-      </a>
-    </li>
-    <li class="integration-list-item">
-      <a href="/preview?type=klarna" class="integration-list-item-link">
-        <div class="title-container">
-          <p class="integration-list-item-title">Klarna - Pay later</p>
-        </div>
-      </a>
-    </li>
-    <li class="integration-list-item">
-      <a href="/preview?type=klarna_account" class="integration-list-item-link">
-        <div class="title-container">
-          <p class="integration-list-item-title">Klarna - Slice it</p>
-        </div>
-      </a>
-    </li>
-    <li class="integration-list-item">
-      <a href="/preview?type=boletobancario_santander" class="integration-list-item-link">
-        <div class="title-container">
```

Comment:
See comment about `boleto` (we can decide whether we include it, I think we should)